browser/resources/resource_ids

```diff
@@ -24,6 +24,7 @@
   "brave/app/theme/brave_unscaled_resources.grd": {
     "includes": [36500],
   },
+  # brave sync 3700
```

Comment:
I think there might have been a rebase issue here. We should include an entry for brave_sync and remove the first_id here https://github.com/brave/brave-core/pull/226/files#diff-00347b03a818f4b0c0e5b8221a7d92a6R15

Reply:
also it's 37000, not 3700